Updated northwind-data.cypher to include AzureOpenAI

HoustonMuzamhindo opened the pull request:
The updated northwind-data has the Cypher if you are using AzureOpenAI instead of the OpenAI endpoint. This is particularly useful when you cannot acccess OpenAI directly due to restrictions in your organisation but you have an AzureOpenAI deployment endpoint ready.

HoustonMuzamhindo committed:
Updated northwind-data.cypher to include AzureOpenAI

The updated northwind-data has the Cypher if you are using AzureOpenAI instead of the OpenAI endpoint. This is particularly useful when you cannot acccess OpenAI directly due to restrictions in your organisation but you have an AzureOpenAI deployment endpoint ready.

zach-blumenfeld commented:
Thank you @HoustonMuzamhindo this is helpful.  Is it possible to have the AzureOpenAI ingest script as a separate file?  That way it is easy for folks to copy past the one that works for them? Maybe like `northwind-data-azureopenai.cypher`? or perhaps a shorter name if you have one in mind?

HoustonMuzamhindo commented:
Hi @zach-blumenfeld . I think that's a better suggestion. I can work on creating the dedicated file.

HoustonMuzamhindo committed:
Create northwind-data-azureopenai.cypher

Created the file dedicated to running the Northwind data ingest using AzureOpenAI

HoustonMuzamhindo committed:
Update northwind-data.cypher

Removed the AzureOpenAI references to keep the file focused on direct access to OpenAI's API.

HoustonMuzamhindo committed:
Update northwind-data-azureopenai.cypher
